mail/1.6/mail.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Jakarta Mail Specification 1.6"
+date: 2019-07-11T16:10:38-08:00
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Servlet defines server-side handling for HTTP requests and responses.
+
+* [Jakarta Mail 1.6 Specification Document](./mail-spec-1.6.pdf) (PDF)
+* [Jakarta Mail 1.6 Specification Document](./mail-spec-1.6.html) (HTML)
+* [Jakarta Mail 1.6 Javadoc](./apidocs)
+* [Jakarta Mail 1.6 TCK](http://download.eclipse.org/ee4j/mail-tck/1.6.0/release/mailtck-1.6.0_10-Jul-2019.zip)
+* Maven coordinates
+ * jakarta.mail:jakarta.mail-api:jar:1.6.4
```

Comment:
What about adding a link to https://search.maven.org/artifact/jakarta.mail/jakarta.mail-api/1.6.4/jar?

Reply:
I suppose we could turn the Maven coordinates text into a link.
Should it go to the search results page, or directly to the download link for the jar file?

Reply:
The search page contains ready to use code snippets for every major build system. On the other hand, that URL might change over time, although it's unlikely.